scripts/context-deploy-benchmark/context-benchmark.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+CONTEXT_AMOUNT=1
+
+run_contexts() {
+  for ((i = 1; i <= CONTEXT_AMOUNT; i++))
```

Comment:
We can deploy the contexts in parallel to reduce bench marking time. AWS load is unpredictable and unknown to us at any given time, so the difference in running parallel vs sequentially will be largely noise